tools/twix/src/panels/map/layers/behavior_simulator.rs

```diff
@@ -39,6 +49,34 @@ impl Layer for BehaviorSimulator {
                 color: Color32::BLACK,
             };
             painter.pose(robot_to_field, 0.15, 0.25, pose_color, pose_stroke);
+
+            let Ok(motion_command): Result<MotionCommand> = self.motion_command.0[player_number].parse_latest() else {
+                continue
+            };
+
+            if let MotionCommand::Walk { path, .. } = motion_command {
+                for segment in path {
+                    match segment {
+                        PathSegment::LineSegment(line_segment) => painter.line_segment(
+                            robot_to_field * line_segment.0,
+                            robot_to_field * line_segment.1,
+                            Stroke {
+                                width: 0.025,
+                                color: Color32::from_rgba_unmultiplied(0, 0, 255, 63),
```

Comment:
I think the colors chosen here or rather the low opacity make the lines quite faint.
Personally I would prefer more contrast between these lines and the green field in the background.
But I would still merge this if you are happy with the current state.

Reply:
Opacity has been increased.